hw_2/HW2_S_Glenn.r

```diff
@@ -24,7 +26,8 @@ directory <- paste("~/Repositories/epi_510/", current_hw_folder, sep = '')
 setwd(directory)
 
 ## 1. Get data from HW 1
-rdsVersion <- readRDS(file = "vipcls.rds")
+vipcls <- readRDS(file = "~/Repositories/epi_510/vipcls.rds")
+
```

Comment:
You were importing the .rds file here as `rdsVersion` but then below were using `vipcls`, so I renamed rdsVersion to vipcls so you're using the correct dataset for this HW.